src/Uno.UI.RuntimeTests/Tests/Uno_Helpers/Given_StartStopEventWrapper.cs

```diff
@@ -0,0 +1,222 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Uno.Helpers;
+using Windows.Devices.Sensors;
+using Windows.Foundation;
+
+namespace Uno.UI.RuntimeTests.Tests.Uno_Helpers
+{
+	[TestClass]
+	public class Given_StartStopEventWrapper
+	{
+		[TestMethod]
+		public void When_Default_Event_Null()
+		{
+			var wrapper = new StartStopEventWrapper<TypedEventHandler<Accelerometer, AccelerometerReadingChangedEventArgs>>(() => { }, () => { });
+			Assert.IsNull(wrapper.Event);
+		}
+
+		[TestMethod]
+		public void When_Add_Handler_Event_Not_Null()
+		{
+			var wrapper = new StartStopEventWrapper<TypedEventHandler<Accelerometer, AccelerometerReadingChangedEventArgs>>(() => { }, () => { });
+			wrapper.AddHandler((s, e) => { });
+			Assert.IsNotNull(wrapper.Event);
+		}
+
+		[TestMethod]
+		public void When_Add_Remove_Event_Null()
+		{
+			void Handler(Accelerometer a, AccelerometerReadingChangedEventArgs e)
+			{
```

Comment:
Codacy found an issue: [Either remove or fill this block of code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5651374)